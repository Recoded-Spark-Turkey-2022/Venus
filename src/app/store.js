// eslint-disable-next-line no-unused-vars
import { configureStore, getDefaultMiddleware } from '@reduxjs/toolkit';
import listingReducer from '../features/listings/listingSlice';

export const store = configureStore({
  reducer: {
    listing: listingReducer,
  },
  // eslint-disable-next-line no-shadow
  middleware: (getDefaultMiddleware) =>
    getDefaultMiddleware({
      serializableCheck: false,
    }),
});
