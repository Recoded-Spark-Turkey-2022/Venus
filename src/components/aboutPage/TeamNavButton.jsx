import React from 'react'

export default function TeamNavButton({children,onClick}) {
  return (
    <button type='button' className='text-[#025594] font-medium rounded-full hover:bg-[#025594] hover:text-[#ffffff] px-[17px] py-[4px]' onClick={onClick}>{children}</button>
  )
}
