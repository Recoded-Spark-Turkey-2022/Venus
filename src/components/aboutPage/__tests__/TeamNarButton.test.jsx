import React from 'react';
import renderer from "react-test-renderer";
import { Provider } from 'react-redux';
import { store } from '../../../app/store';
import { BrowserRouter as Router } from "react-router-dom";
import TeamNavButton from "../TeamNavButton";

jest.mock('swiper/react', () => ({
    Swiper: 'Swiper',
    SwiperSlide: 'SwiperSlide'
  }));
  
  jest.mock('swiper', () => ({
    Pagination: 'Pagination'
  }));
  
  jest.mock('react-i18next', () => ({
    useTranslation: () => {
      return {
        t: (str) => str
      };
    },
  }));

  it(`renders correctly` , () => {
    const tree = renderer.create(
    <Router>
      <Provider store={store}>
        <TeamNavButton />
        </Provider>
    </Router>);
    expect(tree).toMatchSnapshot();
});