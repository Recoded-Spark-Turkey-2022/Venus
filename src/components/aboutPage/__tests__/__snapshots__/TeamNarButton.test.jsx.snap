// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  className="text-[#025594] font-medium rounded-full hover:bg-[#025594] hover:text-[#ffffff] px-[17px] py-[4px]"
  type="button"
/>
`;
